### DISCUSSION TITLE
Is the overlay text only in comination with a frame possible

### DISCUSSION THREAD

**Discussion by Striker62** [photobooth-app/photobooth-app#217] [Q&A] [answered]
Is the overlay text only in combination with a frame possible .
When i disable the frame and enable the text , no text is shown . if I activate the frame and the texet then i have a frame and a text.
Is this standard ? Version 1.2.1

**Comment by mgineer85** [comment, marked as answer]
Works for me.
Please note: If a frame is enabled, the frame is the size of the canvas and so the PosX/PosY values relate to the frame.
If you disable the frame, the actual image as delivered from the camera is the size of the canvas. So if you do not see text, please reduce the PosY value, you should see the text then.

**Reply by Striker62** [reply]
will try thank you
